src/Eloquent/EmbedsRelations.php

```diff
@@ -85,4 +101,17 @@ protected function embedsOne($related, $localKey = null, $foreignKey = null, $re
 
         return new EmbedsOne($query, $this, $instance, $localKey, $foreignKey, $relation);
     }
+
+    /**
+     * Determine if the given key is an embed relationship method on the model.
+     *
+     * @param string $key
+     *
+     * @return bool
+     */
+    public function isEmbeddedRelation($key)
```

Comment:
This new method is not used and not tested. It doesn't seems to be part of the fix.